app/views/account/my_tenants/show.html.slim

```diff
@@ -0,0 +1,18 @@
+.container
+  .row
+    .col-md-12
+      h1= @tenant.name
+      div
+        img.tenant-logo src="https://crunchbase-production-res.cloudinary.com/image/upload/c_lpad,h_120,w_120,f_auto,q_auto:eco/v1459804290/mkxozts4fsvkj73azuls.png"
+        h3 Website:
+        p= link_to span = @tenant.website, "http://#{@tenant.website}", target: "_blank"
+      div
```

Comment:
what about styling? can you make it similar to companies or tenants/show page. I guess we already have a good example of a good styled page. Ask teammates.

Reply:
I did it similar to companies/show page
![tenants-show](https://user-images.githubusercontent.com/28875500/34553592-fb12dc82-f130-11e7-8b81-2fa6029eface.png)